diff --git b/search/edit_distance.go a/search/edit_distance.go
index 95c70db9..861d8ba6 100644
--- b/search/edit_distance.go
+++ a/search/edit_distance.go
@@ -110,7 +110,7 @@ func EditDistanceMaxReuseSlice(a, b string, max int, d []int) (int, bool, []int)
 			lastdiag = olddiag
 		}
 		// after each row if rowmin isn't less than max stop
-		if rowmin >= max {
+		if rowmin > max {
 			return max, true, d
 		}
 	}
