diff --git a/search/levenshtein.go b/search/edit_distance.go
similarity index 87%
rename from search/levenshtein.go
rename to search/edit_distance.go
index dadab252..861d8ba6 100644
--- a/search/levenshtein.go
+++ b/search/edit_distance.go
@@ -18,7 +18,7 @@ import (
 	"math"
 )
 
-func LevenshteinDistance(a, b string) int {
+func EditDistance(a, b string) int {
 	la := len(a)
 	lb := len(b)
 	d := make([]int, la+1)
@@ -51,17 +51,17 @@ func LevenshteinDistance(a, b string) int {
 	return d[la]
 }
 
-// LevenshteinDistanceMax same as LevenshteinDistance but
+// EditDistanceMax same as EditDistance but
 // attempts to bail early once we know the distance
 // will be greater than max
 // in which case the first return val will be the max
 // and the second will be true, indicating max was exceeded
-func LevenshteinDistanceMax(a, b string, max int) (int, bool) {
-	v, wasMax, _ := LevenshteinDistanceMaxReuseSlice(a, b, max, nil)
+func EditDistanceMax(a, b string, max int) (int, bool) {
+	v, wasMax, _ := EditDistanceMaxReuseSlice(a, b, max, nil)
 	return v, wasMax
 }
 
-func LevenshteinDistanceMaxReuseSlice(a, b string, max int, d []int) (int, bool, []int) {
+func EditDistanceMaxReuseSlice(a, b string, max int, d []int) (int, bool, []int) {
 	la := len(a)
 	lb := len(b)
 
diff --git a/search/levenshtein_test.go b/search/levenshtein_test.go
index ef23980e..c4e6aee0 100644
--- a/search/levenshtein_test.go
+++ b/search/levenshtein_test.go
@@ -18,7 +18,7 @@ import (
 	"testing"
 )
 
-func TestLevenshteinDistance(t *testing.T) {
+func TestEditDistance(t *testing.T) {
 
 	tests := []struct {
 		a    string
@@ -38,14 +38,14 @@ func TestLevenshteinDistance(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		actual := LevenshteinDistance(test.a, test.b)
+		actual := EditDistance(test.a, test.b)
 		if actual != test.dist {
 			t.Errorf("expected %d, got %d for %s and %s", test.dist, actual, test.a, test.b)
 		}
 	}
 }
 
-func TestLevenshteinDistanceMax(t *testing.T) {
+func TestEditDistanceMax(t *testing.T) {
 
 	tests := []struct {
 		a        string
@@ -85,7 +85,7 @@ func TestLevenshteinDistanceMax(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		actual, exceeded := LevenshteinDistanceMax(test.a, test.b, test.max)
+		actual, exceeded := EditDistanceMax(test.a, test.b, test.max)
 		if actual != test.dist || exceeded != test.exceeded {
 			t.Errorf("expected %d %t, got %d %t for %s and %s", test.dist, test.exceeded, actual, exceeded, test.a, test.b)
 		}
@@ -107,20 +107,20 @@ var benchmarkTerms = []string{
 	"couchbase",
 }
 
-func BenchmarkLevenshteinDistance(b *testing.B) {
+func BenchmarkEditDistance(b *testing.B) {
 	a := "water"
 	for i := 0; i < b.N; i++ {
 		for _, t := range benchmarkTerms {
-			LevenshteinDistance(a, t)
+			EditDistance(a, t)
 		}
 	}
 }
 
-func BenchmarkLevenshteinDistanceMax(b *testing.B) {
+func BenchmarkEditDistanceMax(b *testing.B) {
 	a := "water"
 	for i := 0; i < b.N; i++ {
 		for _, t := range benchmarkTerms {
-			LevenshteinDistanceMax(a, t, 2)
+			EditDistanceMax(a, t, 2)
 		}
 	}
 }
diff --git a/search/searcher/search_fuzzy.go b/search/searcher/search_fuzzy.go
index f53502ff..92d5b085 100644
--- a/search/searcher/search_fuzzy.go
+++ b/search/searcher/search_fuzzy.go
@@ -242,7 +242,7 @@ func findFuzzyCandidateTerms(ctx context.Context, indexReader index.IndexReader,
 	for err == nil && tfd != nil {
 		var ld int
 		var exceeded bool
-		ld, exceeded, reuse = search.LevenshteinDistanceMaxReuseSlice(term, tfd.Term, fuzziness, reuse)
+		ld, exceeded, reuse = search.EditDistanceMaxReuseSlice(term, tfd.Term, fuzziness, reuse)
 		if !exceeded && ld <= fuzziness {
 			rv.candidates = append(rv.candidates, tfd.Term)
 			rv.editDistances = append(rv.editDistances, uint8(ld))
